src/iterator.rs

```diff
@@ -79,23 +85,16 @@ impl Iterator {
     }
   }
 
-  /// Get the offset for the current position and move the cursor.
-  pub fn offset(&mut self) -> usize {
```

Comment:
I removed this since I couldn't see any use of it. Let me know if I should put it back.

Reply:
looks alright!